integration/nwo/network.go

```diff
@@ -1473,6 +1496,18 @@ func (n *Network) BrokerAddresses(portName PortName) []string {
 	return addresses
 }
 
+// ChaincodeServerAddress returns the address of the chaincode service
+func (n *Network) ChaincodeServerAddress(ccsrv *Chaincode) string {
+	return fmt.Sprintf("127.0.0.1:%d", n.ChaincodeServerPort(ccsrv))
+}
+
+// ChaincodeServerPort returns the named port reserved for the chaincode service.
+func (n *Network) ChaincodeServerPort(ccSrv *Chaincode) uint16 {
+	ccSrvPorts := n.PortsByChaincodeServerID[ccSrv.ID()]
+	Expect(ccSrvPorts).NotTo(BeNil())
+	return ccSrvPorts[ChaincodeServerPort]
+}
+
```

Comment:
This should not be part of the network model; the necessary port can be reserved in the test.